### ISSUE TITLE
host is depracated but...

### ISSUE THREAD

**Issue by selfagency** [imgix/imgix-core-js#36]
my logs are filled with—
```
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
'host' argument is deprecated; use 'domains' instead.
```

but switch `host` to `domains` and you'll get this instead—

```
Error: ImgixClient must be passed a valid host
at new ImgixClient (/Users/daniel/Dev/nylon-juryrig/node_modules/imgix-core-js/dist/imgix-core-js.js:40:15)
at Object.s3 (/Users/daniel/Dev/nylon-juryrig/modules/transforms/imgix.js:18:14)
at imgix (/Users/daniel/Dev/nylon-juryrig/modules/transforms/imgix.js:88:27)
at transImgs (/Users/daniel/Dev/nylon-juryrig/modules/transforms/imgs.js:90:19)
at post (/Users/daniel/Dev/nylon-juryrig/modules/methods/post.js:284:7)
at <anonymous>
```

**Comment by hashknot** [comment]
@selfagency, what version of imgix-core-js are you using?

**Comment by selfagency** [comment]
1.1.1 — i see there's an update

**Comment by hashknot** [comment]
Hi @selfagency, are you still observing this behavior?

**Comment by selfagency** [comment]
no it was fixed in 1.1.1.
